sdk/resourcemanager/Azure.ResourceManager/src/autorest.md

```diff
@@ -14,8 +14,8 @@ mgmt-debug:
   show-request-path: true
 batch:
   - tag: package-common-type-2022-04
-  - tag: package-resources-2022-04
-  - tag: package-management-2022-04
+#   - tag: package-resources-2022-04
```

Comment:
consider to remove comment out lines

Reply:
Sorry, I forgot this. This is only to accelerate my local generation